amqp_job_test.go

```diff
@@ -196,7 +197,7 @@ func TestAMQPJob_createStateUpdateBody(t *testing.T) {
 	job.received = time.Time{}
 	assert.NotContains(t, job.createStateUpdateBody("foo"), "received_at")
 
-	job.Payload().Job.QueuedAt = &time.Time{}
+	job.Payload().Job.QueuedAt = nil
```

Comment:
The `QueuedAt` field is deserialized from JSON, so I'm pretty sure we use `*time.Time` (instead of `time.Time`) as a defensive measure.